### PR TITLE
feat(dato-list): datoSize attribute to change inner input's size

### PR DESCRIPTION
**Pull request by theblushingcrow**

### CODE REVIEW

File: lib/src/list/list.component.ts [hunk 1]
```diff
@@ -85,34 +87,40 @@ export class DatoListComponent extends BaseCustomControl implements OnInit, Cont
...
   /** Client sort comparator */
-  @Input() sortComparator: DatoListSortComparator = defaultClientSortComparator;
+  @Input()
+  sortComparator: DatoListSortComparator = defaultClientSortComparator;
 
   /** Client sort */
-  @Input() sort = true;
-
-  /** available sizes */
-  @Input() datoSize: 'sm' | 'md';
```
[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: I checked beforehand and as far as I could tell in the new dashboard the only person who used dato-list apart from me was Gili, and we talked about ensuring the code doesn't affect his list and syncing when the new version containing this is out. Still want me to change back?

[the remark this answers is not in the document]
Comment (bh86) [reply lib/src/list/list.component.ts]: Great. Please search for usage in the entire project.

[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: I did, only 2 usages - Gili and myself.

File: lib/src/list/list.component.ts [hunk 2]
```diff
@@ -142,6 +150,8 @@ export class DatoListComponent extends BaseCustomControl implements OnInit, Cont
 
   _searchPlaceholder = this.translate.transform('general.search');
 
+  size = 'sm';
```
[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: changed to 
size: string;

File: lib/src/list/list.component.ts [hunk 3]
```diff
@@ -195,8 +205,8 @@ export class DatoListComponent extends BaseCustomControl implements OnInit, Cont
 
     this._data = this.normalizeData(this._data);
     this.initialRun = false;
-    this.datoSize = this.datoSize || 'md';
-    this._listClass = `list-${this.datoSize}`;
+    this.size = this.datoSize || 'sm';
+    this._listClass = `list-${this.size}`;
```
[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: the class is used in the template (to style any select that might occur inside ng-content).

File: lib/src/list/list.component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: Removed.

File: lib/src/list/list.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (theblushingcrow) [reply lib/src/list/list.component.ts]: Removed the size.

**Pull request merged**